voxel_size: 0.01
truncation_distance_factor: 3.0
max_ray_length_m: 5
max_weight: 10000
use_sparsity_compensation_factor: true

object_tracking:
  enable: true
  icp_use_reciprocal_correspondences: true
  icp_max_correspondence_distance: 100
  icp_use_symmetric_objective: false
  icp_max_iterations: 20
  icp_transformation_epsilon: 0.00002
  icp_absolute_mse: 0.000000000001
  icp_euclidean_fitness_epsilon: 0.0001 # 0.01 = improvement of 1% of previous MSE.

meshing:
  update_mesh_every_n_sec: 1.0
  publish_mesh: false
  mesh_filename: "tpp_map.ply"

visualizer:
  enable: true
  camera_intrinsics: [564.3, 0, 480, 0, 564.3, 270, 0, 0, 1]

debug:
  verbose_log: false
